schemas/info_v01.json

```diff
@@ -61,6 +61,11 @@
         "1.0"
       ]
     },
+    "systemAdapter": {
```

Comment:
This exposes system adapter feature. We're currently discussing how to handle this. Please do not sync the schemas for now.

Reply:
Done. Schemas reverted.